src/microbe_stage/Microbe.Contact.cs

```diff
@@ -1756,6 +1768,14 @@ private void CheckStartEngulfingOnCandidate(IEngulfable engulfable)
         else if (EngulfSize > engulfable.EngulfSize * Constants.ENGULF_SIZE_RATIO_REQ && full)
         {
             OnEngulfmentStorageFull?.Invoke(this);
+
+            OnNoticeMessage?.Invoke(this,
+                new SimpleHUDMessage(TranslationServer.Translate("NOTICE_ENGULF_STORAGE_FULL")));
+        }
+        else if (EngulfSize < engulfable.EngulfSize * Constants.ENGULF_SIZE_RATIO_REQ)
+        {
+            OnNoticeMessage?.Invoke(this,
+                new SimpleHUDMessage(TranslationServer.Translate("NOTICE_ENGULF_SIZE_TOO_SMALL")));
         }
```

Comment:
This is also fired when touching friendly species, which shouldn't happen as that implies cannibalism.

Reply:
I'll refactor the engulf check method to return an enum which explains why it fails so the conditions aren't repeated a bunch here.